src/dmd/aggregate.d

```diff
@@ -89,6 +89,9 @@ extern (C++) abstract class AggregateDeclaration : ScopeDsymbol
     /// Only valid for class and struct. TODO: Merge with ClassKind ?
     CPPMANGLE cppmangle;
 
+    /// overridden symbol with pragma(mangle, "...")
+    Identifier mangleOverride;
+    TemplateInstance mangleOverrideTemplate;
```

Comment:
is it worth moving these out of line s.t. it only adds one pointer to `AggregateDeclaration`?

Reply:
Moved this out of line